activities/communication/email-signature-template.md

```diff
@@ -0,0 +1,23 @@
+---
+type: resource
+---
+
+# Email signature template
+
+We use a standard email signature to clearly convey who is communicating and build trust.
```

Comment:
Is this policy or suggestion? I don't think we'd want to enforce this thus opt for weakening the language.

```suggestion
We can use a standard email signature to clearly convey who is communicating and build trust.
```

Reply:
Done.